Sources/BaggageLogging/Baggage+Logger.swift

```diff
@@ -0,0 +1,19 @@
+import Baggage
+import Logging
+
+extension BaggageContext {
+    public var logger: Logger {
+        var logger = self[BaseLoggerKey.self] ?? Logger(label: "BaggageContext")
+        // TODO: Filtering out a specific key to get rid of the logger seems very strange
+        for (key, value) in baggageItems where key.keyType != BaseLoggerKey.self {
+            logger[metadataKey: key.name] = "\(String(describing: value))"
+        }
```

Comment:
Any ideas how on the LogHandler backend that could delegate "to" context so we would not have to create these rendered values each time, but only when the logger actually logs.

Say:

```swift
// assume `context` has a lot of values stored

context.log.trace("is not going to be logged") // ⚠️ oh oh!
// but with current impl we allocated all the string reprs for all the strings!
```

Is there some way we can avoid this with having a LogHandler?

Reply:
This can be a separate ticket and PR :)